### DISCUSSION TITLE
access directly the m3u8 file

### DISCUSSION THREAD

**Discussion by aicon** [bluenviron/mediamtx#1722] [Questions]
I want to access rtsp stream from Ip camera in tivimate on android tv.
So i need the direct access to the m3u8 file to get it working.

e.g I have:
paths:
  ipcam:
    source: rtsp://source

and get:
http://serverip:8888/ipcam/index.m3u8

is this possible?

**Comment by aler9** [comment]
Hello, yes, it's perfectly possible to access the HLS stream without using the browser. It's even supported natively by VLC and Safari.

[a comment by an automated account is not shown]
